### PR TITLE
#24065 Sending variantName params when edit a Contentlet

### PR DESCRIPTION
**Pull request by freddyucv**
We need to sent the variantName when we are render a Contentlet into a Container, this happend after edit a Contentlet inside the Edit Page Portlet

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts [hunk 1]
```diff
@@ -9,7 +10,7 @@ import { DotContainerContentletService } from './dot-container-contentlet.servic
 
 import { DotPageContent } from '../../shared/models';
 
-describe('DotContainerContentletService', () => {
+fdescribe('DotContainerContentletService', () => {
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts:13]: remove

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts [hunk 2]
```diff
@@ -91,7 +93,52 @@ describe('DotContainerContentletService', () => {
...
+            .subscribe();
+        httpMock.expectOne(`v1/containers/content/2?containerId=1&pageInode=2&variantName=Testing`);
+    });
+
     afterEach(() => {
         httpMock.verify();
     });
+
+    afterEach(() => {
+        window.sessionStorage.removeItem('variantName');
```
Comment (zulqarnainvd) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts:142]: can we add this line in above `afterEach` Cycle and remove this extra  `afterEach`?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts (hunk 1 above)
Comment (freddyucv) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts]: done https://github.com/dotCMS/core/pull/24159/commits/0a5d748ba5cc237b7e05d6d63b451e9142b5620c#diff-2288713eef0847dd9f6a81ce4c0599266039202d94f3d30a012316ae93856e2eR13

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts (hunk 2 above)
Comment (freddyucv) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts:139]: done https://github.com/dotCMS/core/pull/24159/commits/0a5d748ba5cc237b7e05d6d63b451e9142b5620c#diff-2288713eef0847dd9f6a81ce4c0599266039202d94f3d30a012316ae93856e2eL141

**Review by alfredo-dotcms — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

**Pull request merged**